### PR TITLE
Jr/berks page

### PR DESCRIPTION
**Pull request by jeffrummel**
Still WIP. Not ready for review

### CODE REVIEW

**Review by carlmjohnson — APPROVED** [review]
image

First name / Last name are sort tight on mobile. Maybe stack them?

Looks great otherwise. Really nice.

**Comment by carlmjohnson** [comment]
I've updated the email server, so now you just need to add a hidden field with `<input type="hidden" name="berks_county" value="1" />` to sign someone up.

File: content/berks/_index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+takeover = """
+{{<featured/main-takeover
+  title = "Help us write the future of journalism in Berks County."
+  dek = "Join our historic effort to invigorate local news."
+  image = "2023/09/01jf-h6pe-1sks-18dz.jpeg"
+  signupHed = "Join our mailing list "
+  signupDek = "Sign up to receive a weekly review of articles from Spotlight PA and occasional updates about our planning project in Berks County"
+  gaCategory = "newsletters:palocal"
+  gaForm = "newsletters:palocal-full"
+  hiddenField = "newsletter/hidden-input.html"
```
Comment (carlmjohnson) [inline content/berks/_index.md:22]: I would clone newsletter/hidden-input.html to newsletter/berks-county-hidden.html and add `<input type="hidden" name="berks_county" value="1" />` to the end of the clone. That will sign people up for Breaking News, Week In Review, and Berks County, which should be good.

Comment (jeffrummel) [reply content/berks/_index.md]: Thanks!

Updated: https://github.com/spotlightpa/poor-richard/pull/468/commits/e4927d6c5fa7942594a14e4ae2f611f7325411ec

**Pull request merged**